messaging-client-databus/src/main/java/com/inmobi/databus/partition/ClusterReader.java

```diff
@@ -128,4 +128,11 @@ public MessageCheckpoint buildStartPartitionCheckpoints() {
     return new DeltaPartitionCheckPoint(
         dataWaitingReader.buildStartPartitionCheckpoints());
   }
+
+  @Override
+  public Long getReaderBackLog() throws IOException {
+    Long pendingSize = 0l;
```

Comment:
If it is not intended to implement within ClusterReader, throw UnsupportedOperationException.